homework/style.css

```diff
@@ -1,3 +1,71 @@
+header {
```

Comment:
The color contrast between texts and backgrounds (for repo info and contributors) is too low (the text should be darker). This can make it hard for some people to read it (for example vision impaired or when using a crappy screen).